blueprints/locale/index.js

```diff
@@ -3,23 +3,20 @@
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
 
+var locales     = require('formatjs-extract-cldr-data/lib/locales');
```

Comment:
Should we be exporting this from formatjs-extract-cldr-data?  I need a way to validate the locale